typescript-methods/main.ts

```diff
@@ -0,0 +1,67 @@
+const heroes = ['thor', 'loki', 'odin', 'hulk'];
+let randomNumber: number = Math.random();
+
+const heroesNumber: number = heroes.length;
+
+randomNumber = randomNumber * heroesNumber;
+
+const randomIndex = Math.floor(randomNumber);
+
+console.log(randomIndex);
+
+console.log(heroes[randomIndex]);
+
+const library = [
+  {
+    title: 'The Great Gatsby',
+    author: ' F. Scott Fitzgerald',
+  },
+  {
+    title: 'To Kill a Mockingbird',
+    author: 'Harper Lee',
+  },
+  {
+    title: '1984',
+    author: 'George Orwell',
+  },
+];
+const lastBook = library.pop();
+console.log('Last book: ', lastBook);
+const firstBook = library.shift();
+console.log('First book: ', firstBook);
+const js = {
+  title: 'JavaScript for Impatient Programmers',
+  author: 'Dr. Axel Rauschmayer',
+};
+const css = {
+  title: 'CSS Secrets',
+  author: 'Lea Verou',
+};
+library.push(js);
+library.push(css);
+library.splice(0, 1);
```

Comment:
This should be `library.splice(1, 1)`. We want to start at index 1 rather than index 0